chall01/gmolin.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+import sys
+
+CODE = {'A': '.-',     'B': '-...',   'C': '-.-.', 
```

Comment:
I suggest googling `PEP8` it can help with the variable convention name, `code` should be lowercase.